### PR TITLE
Add Telugu support

### PR DESCRIPTION
**Pull request by Shubhamjain27**
Language Models and Classifier for Telugu

https://github.com/Shubhamjain27/nlp-for-telugu

### CODE REVIEW

**Comment by goru001** [comment]
Thanks @Shubhamjain27 for contributing. Great work!

It'd be great if you can make a few changes according to the following comments (in the order as I observe them):

1. In order to maintain consistency with other repositories, it would be great if you can upload and make the datasets public on Kaggle, with appropriate usage guidelines and credits. 

2. In your PR, add your repo and the results in the "Repositories containing models used in iNLTK" table in inltk's README and docs/api_docs.md

3. (Minor) It'd be great if you can remove unnecessary files (like .ipynb checkpoints etc) from your repo

4. In your language model training notebook, I see that you're creating your train-test split and training your model. But since we don't have that same train-test split, it'll be very difficult for me (or anyone else) to reproduce your results. So it would be great if you first split your Dataset (use seeds while splitting), and make that particular Train/Test set available for others to reproduce/benchmark/experiment. Making these datasets available on Kaggle is a good practice that has been followed in other repos of iNLTK.

5. It seems you are concatenating "heading of wikipedia article" and "wikipedia article" by space. I guess using newline will be a better choice here if at all you want to also use the headings. Just using wikipedia articles should also be fine.

6. Though the fastai api has changed over the course of time, so I'm not sure but I guess when you're creating `language_model_learner` object, you've to pass the parameter `pretrained=False` to ensure you're starting training from scratch. It seems your LM model training started from pretrained English LM. Do check this, Let me know if I'm wrong here.

7. I guess lot of above problems exist in Transformer LM, classification notebooks as well, do correct.

Thanks for all your efforts. Do let me know if I missed something.

**Comment by Shubhamjain27** [comment]
Thanks @goru001 for suggesting the changes. I have made all the changes.

**Comment by goru001** [comment]
Thanks @Shubhamjain27 for all your efforts. Had a quick look over your notebooks, 
1. I see in the ULMFiT LM notebook that the LM was trained only for 1 epoch, is that correct? 
2. Can you also benchmark model's performance on this [dataset](https://www.kaggle.com/sudalairajkumar/telugu-nlp?) ?

**Comment by Shubhamjain27** [comment]
Thanks @goru001 for your kind feedback. 

1. I trained it for multiple epochs earlier, however the accuracy was decreasing by a small margin over time, and hence in the final version trained it for one. 
2. I have benchmarked it and achieved 92% accuracy. I have added the notebook in the repository. 

Thank you again!

**Review by goru001 — APPROVED** [review]
Looks good
